src/src.pro

```diff
@@ -56,6 +56,7 @@ CONFIG(release, debug|release) {
 include(../version.pri)
 
 # Qt defines
+DEFINES += QT_DISABLE_DEPRECATED_BEFORE=0x050e00
 DEFINES += QT_DEPRECATED_WARNINGS
```

Comment:
This has no effect now, you might want to remove it.
https://doc.qt.io/qt-5/qtglobal.html#QT_DEPRECATED_WARNINGS

Reply:
> 
> This has no effect now, you might want to remove it.
> https://doc.qt.io/qt-5/qtglobal.html#QT_DEPRECATED_WARNINGS

Removed.